src/components/feedback/Spinner.js

```diff
@@ -0,0 +1,33 @@
+import classnames from 'classnames';
+
+import { SpinnerSpokesIcon } from '../icons';
+
+/**
+ *
+ * @typedef SpinnerProps
+ * @prop {'sm'|'md'|'lg'} [size='sm']
```

Comment:
This is called out in the pattern-library documentation changelog, but I've changed default sizing from medium to small because in most of our usage of spinners that's the size we're using.